src/components/ScholarshipDetailCard.jsx

```diff
@@ -32,13 +115,60 @@ export default function ScholarshipDetailCard({ scholarship }) {
         {description}
       </Typography>
 
-      <Button
+      <Box className={classes.groupCellWithoutHeader}>
+        <ScholarshiDetailCardCell
+          label="Deadline"
+          text={deadline?.toLocaleDateString()}
+        />
+        <ScholarshiDetailCardCell
+          label="Award Amount"
+          text={amount?.toString()}
+        />
+        <ScholarshiDetailCardCell label="State" text={states?.join(', ')} />
+      </Box>
+
+      <Box className={classes.eligibilitiesWrapper}>
+        <Typography
+          variant="h6"
+          component="h4"
+          className={classes.sectionHeader}>
+          Eligibility Requirements
+        </Typography>
+
+        <ScholarshiDetailCardCell label="GPA" text={eligibility?.GPA} />
+        <ScholarshiDetailCardCell
+          label="Demographic"
+          text={eligibility?.ethnicities}
+        />
+        <ScholarshiDetailCardCell label="Majors" text={eligibility?.majors} />
```

Comment:
I suggest [transforming the keys and lists](https://reactjs.org/docs/lists-and-keys.html) in `eligibility` to detail cells instead.